mmf/modules/encoders.py

```diff
@@ -125,6 +125,15 @@ def __init__(self, config: Config, *args, **kwargs):
         self.lc.bias.data.copy_(torch.from_numpy(bias))
         self.out_dim = out_dim
 
+        def state_dict_key_override(state_dict, prefix, *args, **kwargs):
+            old_prefix = prefix + "module."
```

Comment:
We shouldn't hardcode this IMO. In case there are other prefixes in future that we want to replace how can we handle it?

Reply:
Right now, whenever there is code changes, we have to go into the top level format_state_key to perform state_key replacement. https://github.com/facebookresearch/mmf/search?q=format_state_key

Similarly to how we did it before, we'd have to modify the code to add additional prefixes that we wish to override.